File: SEIST/polyglot/nim/seis_motion_policy.nim
type
  SeisMotionPolicy* = object
    supportsReducedMotion*: bool
    mobileParticleLimit*: int
    desktopParticleLimit*: int
    continuousSceneLimit*: int

const foundationMotionPolicy* = SeisMotionPolicy(
  supportsReducedMotion: true,
  mobileParticleLimit: 24,
  desktopParticleLimit: 64,
  continuousSceneLimit: 1
)

File: SEIST/polyglot/pascal/SeisMotionPolicy.pas
unit SeisMotionPolicy;

interface

type
  TSeisMotionPolicy = record
    SupportsReducedMotion: Boolean;
    MobileParticleLimit: Integer;
    DesktopParticleLimit: Integer;
    ContinuousSceneLimit: Integer;
  end;

const
  FoundationMotionPolicy: TSeisMotionPolicy = (
    SupportsReducedMotion: True;
    MobileParticleLimit: 24;
    DesktopParticleLimit: 64;
    ContinuousSceneLimit: 1
  );

implementation

end.
